### PR TITLE
[3.0] Fix: module references

### LINKED ISSUE #76: marked node_module missing + wrong reference
With 3.0, I'd expect the marked-element to download marked into the node_modules root folder. right now it's not installed. On top of that, the path reference needs to point to the marked folder in the root.

### PR DESCRIPTION
**Pull request by stramel**
Fixes #76 

Changes:

- ~~`marked-element` is contained within `@polymer` so we need to move up an extra folder to get to `@webcomponents` (Also, add `@webcomponents` to the webcomponentsjs path.) (https://github.com/Polymer/polymer-modulizer/issues/174)~~
- Removed the extra file from the previous work around in HTML Imports for loading a JS file. Swapped to using the import inline. (https://github.com/Polymer/polymer-modulizer/issues/182)
- Removed `'use strict';` since this is always enabled in modules now. (https://github.com/Polymer/polymer-modulizer/issues/181)
- Added the missing `marked` dependency from conversion. (https://github.com/Polymer/polymer-modulizer/issues/174)
- Added the `paper-styles` dev depenency for demo. (https://github.com/Polymer/polymer-modulizer/issues/176)
- Added `node_modules` to `.gitignore` (https://github.com/Polymer/polymer-modulizer/issues/183)
- Update Tests to use "legacy" `wct-legacy-browser` and all references (Thanks @usergenic)
- Tests aren't working with `polymer test --npm` (@usergenic any thoughts? https://github.com/Polymer/polymer-cli/issues/858)
- Update `.travis.yml` to run `polymer test --npm` and install using `yarn` (https://github.com/Polymer/polymer-modulizer/issues/186)
- Added the `yarn.lock`.
- Switched to a SPDX license. (I believe this is the one that was referenced) (https://github.com/Polymer/polymer-modulizer/issues/185)

Other things to note: 
- iron-component-page isn't working (No `3.0-preview` branch, https://github.com/Polymer/polymer-modulizer/issues/180)

@rictic @e111077 PTAL

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,9 +19,17 @@
     "type": "git",
     "url": "git://github.com/PolymerElements/marked-element.git"
   },
-  "license": "https://polymer.github.io/LICENSE.txt",
+  "license": "BSD-3-Clause",
```
Comment (TimvdLippe) [inline package.json:22]: I dont think we can make this change here, we would need to update every repository at once if we need to.

Comment (stramel) [reply package.json:22]: Okay, may be worth doing since NPM throws a warning about non-SPDX license. I can revert this if y'all would like.

**Comment by justinfagnani** [comment]
Right now, I think we should be making changes to the mast branch of elements, or to polymer-modulizer. If we do a mass conversion again, we'd lose these changes, and I don't think we're quite ready for a real cut-over to 3.0.

**Comment by justinfagnani** [comment]
@stramel this is a great set of things to fix in modulizer! I'm trying to port some of the issues over there, feel free to add more if I miss anything.

**Comment by justinfagnani** [comment]
Oh wait, you already mostly did that. Sorry, jetlag brain today :)

**Comment by stramel** [comment]
@justinfagnani I agree these changes should be fixed upstream in the polymer-modulizer. It was a good learning experience by seeing what was missing from the auto-conversion. Would there be any benefit to merging/keeping these around just for people to use until these things are addressed? If not, feel free to close :)

**Comment by justinfagnani** [comment]
I don't think so, if we merged they would just be blown away by the next conversion. We can still reference this closed PR too. Thanks for doing this!

**Pull request closed without merging**